client/lib/media/test/actions.js

```diff
@@ -69,6 +69,7 @@ describe( 'MediaActions', function() {
 						return {
 							get: mediaGet.bind( [ siteId, mediaId ].join() ),
 							update: mediaUpdate.bind( [ siteId, mediaId ].join() ),
+							edit: mediaUpdate.bind( [ siteId, mediaId ].join() ),
```

Comment:
They do almost the same action with the difference that `edit()` accepts a `media` field to update the file of the media post. I think it's fine for testing purpose.